conda_forge_metadata/artifact_info/info_json.py

```diff
@@ -135,24 +135,45 @@ def info_json_from_tar_generator(
         elif path.name == "about.json":
             data["about"] = json.loads(_extract_read(tar, member, default="{}"))
         elif path.name == "conda_build_config.yaml":
+            assert data["conda_build_config"] == {}
             data["conda_build_config"] = YAML.load(
                 _extract_read(tar, member, default="{}")
             )
-        elif path.name == "files":
-            files = _extract_read(tar, member, default="").splitlines()
```

Comment:
some _very_ old packages (like `python-3.5.1-0`) don't have `paths.json`, but most (at least a package that was 6years old already had it) packages should have it.
should i still implement a fallback for `files` if `paths.json` doesn't exist or is it okay like this?

Reply:
Yes we need a fall back to files if paths.json is not found.